tasks/setup-Debian.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensure that gcc is available.
+  apt:
+    name: gcc
+    state: present
+    update_cache: yes
```

Comment:
I typically let playbooks set `update_cache` in their own pre_tasks rather than in a role itself. I would rather not add this parameter here for consistency's sake.

Reply:
Done

Reply:
Maybe a better solution is to remove this step entirely. As I said at the top, this is the only step that is specific to Debian. Subject to testing/debugging.